src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitScm.java

```diff
@@ -341,15 +518,40 @@ public GitLfsPull(GitScm git, String path) {
     }
 
     public static class MergeBeforeBuild extends Behaviour {
-        private final Control txtMergeRemote = control("mergeRemote");
-        private final Control txtMergeTarget = control("mergeTarget");
-        private final Control selMergeStrategy = control("mergeStrategy");
-        private final Control selFastForwardMode = control("fastForwardMode");
+        private final Control txtMergeRemote = control("options/mergeRemote");
+        private final Control txtMergeTarget = control("options/mergeTarget");
+        private final Control selMergeStrategy = control("options/mergeStrategy");
+        private final Control selFastForwardMode = control("options/fastForwardMode");
```

Comment:
Will this work for older git plugin releases? If not, please use both paths - newest first - to preserve the compatibility.

Reply:
In that case is not necessary to preserve compatibility. I did test with version 2.5.X (first time this behaviour is included), 2.3.2 and 2.6.0. The correct path has been always `options/XXX`.